src/BinSkim.Sdk/MetadataConditions.cs

```diff
@@ -10,6 +10,7 @@ public static class MetadataConditions
         public static readonly string ImageIsNotExe = SdkResources.MetadataCondition_ImageIsNotExe;
         public static readonly string ImageIsNotMachO = SdkResources.MetadataCondition_ImageIsNotMachO;
         public static readonly string CouldNotLoadPdb = SdkResources.MetadataCondition_CouldNotLoadPdb;
+        public static readonly string ImageIsArmBinary = SdkResources.MetadataCondition_ImageIsArmBinary;
```

Comment:
>public static readonly string ImageIsArmBinary = SdkResources.MetadataCondition_ImageIsArmBinary; [](http://example.com/codeflow?start=8&length=97)

Add a MetadataConditions entry for ARMBinaries. This is separate as changing the previous user facing string could confuse customers by changing details in the results.